src/Cli/Microsoft.DotNet.Cli.Utils/BuiltInCommand.cs

```diff
@@ -63,7 +63,7 @@ public CommandResult Execute()
                 var taskOut = _stdOut.BeginRead(new StreamReader(outStream));
                 var taskErr = _stdErr.BeginRead(new StreamReader(errorStream));
 
-                int exitCode = _builtInCommand(_commandArgs.ToArray());
+                int exitCode = _builtInCommand([.. _commandArgs]);
```

Comment:
Not a fan of this one (or below examples)

Reply:
After some discussion, I've decided to keep this as it follows suit with newer language conventions. I don't know how widely adopted they are, though. Collection manipulation has been evolving in the past few years, so I feel these kinds of statements will become more commonplace going forward.